core/modules/replica/tools/qserv-replica-job-replicate.cc

```diff
@@ -20,128 +20,29 @@
  * see <http://www.lsstcorp.org/LegalNotices/>.
  */
 
-/// qserv-replica-job-replicate.cc is a single job Controller application
-/// which is meant to run the corresponding job.
+/**
+ * qserv-replica-job-replicate.cc is an application which analyzes the replication
+ * level for all chunks of a given database family and bring the number of replicas
+ * up to the explicitly specified (via the corresponding option) or implied (as per
+ * the site's Configuration) minimum level. Chunks which already have the desired
+ * replication level won't be affected by the operation.
+ */
```

Comment:
Repeating the same comment does not make much sense, just say `@see ReplicateApp` (or get rid of this file entirely).

Reply:
Good point